docs/configuration.md

```diff
@@ -19,6 +19,7 @@ For example, the config object may look similar to the following:
 | Field Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | allowCookies | Boolean | `false` | Enable the web client to set and read two cookies: a session cookie named `cwr_s` and a user cookie named `cwr_u`.<br/><br/>`cwr_s` stores session data including an anonymous session ID (uuid v4) created by the web client. This allows CloudWatch RUM to compute sessionized metrics like errors per session.<br/><br/>`cwr_u` stores an anonymous user ID (uuid v4) created by the web client. This allows CloudWatch RUM to count return visitors.<br/><br/>`true`: the web client will use cookies<br/>`false`: the web client will not use cookies. |
+| aws:releaseId | string | `1.0.0` | The application release id used to find a matching source map, if any. Must match regex `^[a-zA-Z0-9_\-:/\.]+$` |
```

Comment:
blockers: 
1. Can we also mention the max length of 200 limit in the docs as well?
2. This field should not have a default value given customers will also have to configure the directory in S3 when they add their source map files. This field should be undefined and not added to the event metadata unless specifically set by the customer.